testclients/bug25714.c

```diff
@@ -13,64 +13,52 @@
    along with this program; if not, write to the Free Software
    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA */
 
+#include <assert.h>
```

Comment:
Non-minimum diff - looks like this file was clang-format'ed, some changes go against MySQL coding standard